shared/gh/js/gh.calendar.js

```diff
@@ -41,13 +41,13 @@ define(['gh.core', 'moment', 'clickover'], function(gh, moment) {
         {
             'name': 'lent',
             'label': 'Lent',
-            'start': '2015-01-13',
+            'start': '2015-01-15',
```

Comment:
I've created a ticket at https://github.com/CUL-DigitalServices/grasshopper-ui/issues/124 to make this configurable through the admin UI.